### ISSUE TITLE
Edit Question: ua_ruhr_v1.1.0-beta.1/content-classification/data-dataset/description

### ISSUE THREAD

**Issue by smohex** [FDM-UARuhr/rdmo-catalog-uaruhr#53]
**Describe the request**  
Addition of the help text description about the time/period of occurrence of the dataset

**Propose what exactly should be changed**  
Add at the end of the help text:

*help text ger*  
"Bitte erläutern Sie ergänzend, ob es sich um einen festen, zu einem bestimmten Zeitpunkt gespeicherten Datensatz handelt oder ob dieser dynamisch also laufend über einen bestimmten Zeitraum entsteht."
* help text en*
"Please also explain whether this is a fixed data set stored at a certain point in time or whether it is created dynamically over a certain period of time."

**From the following list delete everything that is NOT part of the content of your proposal. Once a subject of change is approved it will be marked accordingly in the list.**  

- [x] help text (en and ger)

**Comment by jmfrenzel** [comment]
https://github.com/rdmorganiser/rdmo-catalog/issues/77

**Comment by jmfrenzel** [comment]
solved upstream via https://github.com/rdmorganiser/rdmo-catalog/pull/85
